### PR TITLE
week-1 exercise done

### PR DESCRIPTION
**Pull request by sahil-tgs**

### CODE REVIEW

File: exercises/week 1/solutions/sahil-tgs/chapter1/ecc.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
+from unittest import TestCase
+
+
+
+
+class FieldElement:
+
+
+
+
```
Comment (rajarshimaitra) [inline exercises/week 1/solutions/sahil-tgs/chapter1/ecc.py:10]: too many empty lines.. is this intentional?

**Review by rajarshimaitra — COMMENTED** [review]
ACK 28ea5257446b3a12365f40c0f3a2cbf81d94567f

**Comment by rajarshimaitra** [comment]
you can drop the satoshi folder from the PR..

File: exercises/week 1/solutions/sahil-tgs/chapter1/ecc.py (hunk 1 above)
Comment (sahil-tgs) [reply exercises/week 1/solutions/sahil-tgs/chapter1/ecc.py]: No, the code formatter must have glitched out or something. I have reduced the unnecessary lines and have made a 2nd commit.

Comment (rajarshimaitra) [reply exercises/week 1/solutions/sahil-tgs/chapter1/ecc.py]: Still one more extra lines.. but okay..

**Review by rajarshimaitra — APPROVED** [review]
ACK 9273f69399e4f215bd862c480a867bc4bae8467a

**Pull request merged**